docs/structure.rst

```diff
@@ -27,14 +27,35 @@ Overview of directories and files
 
 -  ``libraries/``: libraries of Dedukti files (see GNUmakefile)
 
+- ``src/backbone/``: miscellaneous modules and libraries
```

Comment:
the directory lplib is missing now (not fixed in previous PR)

Reply:
This file has not been updated for quite some time and many files are missing, have been deleted or renamed.

Missing files: builtin, inductive, package, pratt, unif_rule, sig_state

Removed files: typing

Thanks to make sure that every existing file is listed and that every listed file exists.

Reply:
Thanks to update CONTRIBUTING.md as well to indicate that this file should be kept up to date.